source/draw/gpu/opengl/backend/GLExtensions.ooc

```diff
@@ -14,12 +14,24 @@ GLExtensions: class {
 	eglCreateImageKHR: static Func(Pointer, Pointer, UInt, Pointer, Int*) -> Pointer
 	eglDestroyImageKHR: static Func(Pointer, Pointer)
 	glEGLImageTargetTexture2DOES: static Func(UInt, Pointer)
+	eglCreateSyncKHR: static Func(Pointer, UInt, Int*) -> Pointer
+	eglDestroySyncKHR: static Func (Pointer, Pointer) -> Bool
+	eglClientWaitSyncKHR: static Func (Pointer, Pointer, Int, ULong) -> Bool
+	eglDupNativeFenceFDANDROID: static Func(Pointer, Pointer) -> Int
 	_initialized := static false
 	initialize: static func {
 		if (!This _initialized) {
 			This eglCreateImageKHR = This _load("eglCreateImageKHR") as Func(Pointer, Pointer, UInt, Pointer, Int*) -> Pointer
 			This eglDestroyImageKHR = This _load("eglDestroyImageKHR") as Func(Pointer, Pointer)
 			This glEGLImageTargetTexture2DOES = This _load("glEGLImageTargetTexture2DOES") as Func(UInt, Pointer)
+			This eglCreateSyncKHR = This _load("eglCreateSyncKHR") as Func(Pointer, UInt, Int*) -> Pointer
+			This eglDestroySyncKHR = This _load("eglDestroySyncKHR") as Func (Pointer, Pointer) -> Bool
+			This eglClientWaitSyncKHR = This _load("eglClientWaitSyncKHR") as Func (Pointer, Pointer, Int, ULong) -> Bool
+			version(!windows) {
+				//For some reason this function can't be loaded with eglGetProcAddress so we load it with dlsym instead
+				RTLD_DEFAULT := null //This should be defined in dlfcn.h
```

Comment:
Yes, I remember something similar from when I sorted out the mess of external C includes in this repo.

If this works satisfactory then let's be happy with this. I've made an issue #1843 in case we have time to think about it further later.